### ISSUE TITLE
CLI failing execution when called from a script executed by cloud-init

### ISSUE THREAD

**Issue by marcociarrocca** [scaleway/scaleway-cli#1952]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

## Command attempted

```bash
scw init secret-key=${scw-secret-key} send-telemetry=false install-autocomplete=false with-ssh-key=false -c "/root/.config/scw/config.yaml"
scw instance ip create -c "/root/.config/scw/config.yaml" project-id=${project-id} server=${server-id} zone=${scw-zone}
```

### Expected Behavior

A public IP is created and attached to the server instance.

### Actual Behavior

When executed manually, everything works correctly.
When executed by a script called by cloud-init, the following error is thrown by both commands: `Scaleway-sdk-go: cannot read config file: read .: is a directory`
When omitting the `-c` flag, the following error is thrown by the `scw init` command: `Open .: is a directory`; the `ip create` command fails with the same error reported above.
Omitting the `-c` flag and specifying the `SCW_CONFIG_PATH` env variable shows no difference.
The config file seems to get created correctly (and on the right path, when specified), while the `ip create` command always fails (does not create the IP).
Executing the commands either manually or via a manually-executed script (after SSHing into the instance) works correctly.

## More info

```
scw version:
Version    2.3.1
BuildDate  2021-06-08T11:54:02
GoVersion  go1.16.4
GitBranch  new-release
GitCommit  07dbaebc
GoArch     amd64
GoOS       linux

uname:
Linux 5.10.28-scaleway #1 SMP x86_64 GNU/Linux
```

**Comment by kerberjg** [comment]
I've been trying to read the source code of both `scaleway-cli` and `scaleway-sdk-go` (since the latter seems to be mentioned by the error) and I've found that what happens is essentially this:

- scaleway-cli tries to get the config file path from 3 sources, in order of priority: -c flag, `'$HOME/.config/scw/config.yaml'`, and finally `$SCW_CONFIG_FILE` https://github.com/scaleway/scaleway-cli/blob/d8a025334859e6ac812af1ee7746ca4c6c8c61da/internal/namespaces/config/commands.go#L262, https://github.com/scaleway/scaleway-cli/blob/d8a025334859e6ac812af1ee7746ca4c6c8c61da/internal/core/context.go#L146
- One of those is being interpreted as `.` since that's what seems to be the extrapolated path
- `scaleway-sdk-go` tries to open it, spitting the `cannot read config file` error; https://github.com/scaleway/scaleway-cli/blob/d8a025334859e6ac812af1ee7746ca4c6c8c61da/internal/namespaces/config/commands.go#L263, https://github.com/scaleway/scaleway-sdk-go/blob/043be10c857c516a5dadd1b277eceb2207eeacab/scw/config.go#L203

Furthermore, the -c flag is being extrapolated in the bootstrap.go file, but there's a mistake; 

https://github.com/scaleway/scaleway-cli/blob/5332ec180848faebf18fb9b9af3d6c872d925da0/internal/core/bootstrap.go#L228

The references to profileFlag and configFileFlag are swapped.
I wonder if this is of significance given that they're being read (?) correctly earlier on.

https://github.com/scaleway/scaleway-cli/blob/5332ec180848faebf18fb9b9af3d6c872d925da0/internal/core/bootstrap.go#L79

Pinging @jerome-quere for relevance

**Comment by Codelax** [comment]
Broken config flag has been fix with #2476. It will be included in next release

**Comment by Codelax** [comment]
Also, `scw init secret-key=${secret-key}` will still open interactive prompt to fill access key even tho it is not needed. I will disable this interactive prompt and it should be good
